apps/astarte_appengine_api/test/support_v2/generators/mapping.ex

```diff
@@ -0,0 +1,91 @@
+defmodule Astarte.Test.Generators.Mapping do
+  use ExUnitProperties
+
+  alias Astarte.Core.Mapping
+
+  defp endpoint(%{aggregation: aggregation, prefix: prefix}) do
+    # TODO: thinking about it
```

Comment:
Make this TODO a little more explicit so that the next person working on this knows what there's to think about